Documentation update (#191)
acrolinx/platform-api: README.md

    }
    ```

## Create an API token

Creating an API token is easy:

1. Sign in to Acrolinx.
2. Go to your ‘Settings'.
3. Click CREATE API TOKEN.
4. Copy the API token.

**Note**
After you generate an API token, save it somewhere secure (like a password manager) so that you can access it again. Once you leave the Settings page, the API token will no longer appear on screen.

## Signature

A signature is a string that identifies an integration.

> Inclue the signature as a header in every request you send to the Acrolinx API:

Reviewer: ```suggestion
Include the signature as a header in every request you send to the Acrolinx API:
```

```HTTP
X-Acrolinx-Client: YOUR_SIGNATURE; VERSION_NUMBER
```

The `VERSION_NUMBER` should follow the following pattern `X.X.X.X` like `1.0.0.1`.

To get [certified](https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/checklist.md), your integration

> must send the [version and build number](https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/project-setup.md#version-information).

Reviewer: ```suggestion
needs to send the [version and build number](https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/project-setup.md#version-information).
```

Maintaining your integration in production is much easier with a version and build number.

> ## API Reference

Reviewer: ```suggestion
## API reference
```
